KuduSync.NET/Program.cs

```diff
@@ -7,20 +7,53 @@ namespace KuduSync.NET
 {
     class Program
     {
+        public const string AppOfflineFileName = "app_offline.htm";
+        private const string AppOfflineFileContent =
```

Comment:
you might consider adding this as an embedded resource. That's what I did in the Functions runtime: https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script.WebHost/Resources/app_offline.htm

Reply:
@mathewc while I agree that embedded resource is cleaner, I don't think it's worth changing. It's not like we'll be constantly modifying that page. It'll like never change. msdeploy has this exact code, BTW. Let's leave it this way.